components/create_comment/create_comment.test.tsx

```diff
@@ -1488,7 +1537,7 @@ describe('components/CreateComment', () => {
     ), (instance) => instance.state().draft.message);
```

Comment:
Typescript is failing because of `Parameter 'instance' implicitly has an 'any' type`. I used `instance: CreateCommentClass` but no luck

Any idea how to solve this.

Reply:
I think the name of `instance` is misleading. Looking at `testComponentForLineBreak`, that argument is actually a wrapper as returned from `shallow`, so it should be a `ShallowWrapper` of some sort. Perhaps `ShallowWrapper<CreateCommentClass>` would work there?